values/cloudnative-pg/cloudnative-pg.gotmpl

```diff
@@ -0,0 +1,9 @@
+{{- $v := .Values }}
+{{- $es := $v.apps | get "cloudnative-pg" }}
+
+limits:
+  cpu: 100m
+  memory: 200Mi
```

Comment:
Let's increase limits
  cpu: 500m
  memory: 512Mi

Reply:
Fixed in: https://github.com/redkubes/otomi-core/pull/1137/commits/19d1d8dde426694bffee41dd3b0a56cc7fabb904